package.json

```diff
@@ -43,6 +43,7 @@
     "got": "^6.6.3",
     "heroku-cli-util": "^6.0.15",
     "heroku-run": "^3.4.3",
+    "opn": "^4.0.2",
```

Comment:
@maxbeizer I don't think it's necessary including a new package for this as this is something we can do using `heroku-cli-util` with `cli.open`. For example: https://github.com/heroku/heroku-orgs/blob/a84ad8395017177452678182cfac445d7fa20844/commands/orgs/open.js.